### DISCUSSION TITLE
job['enqueued_at'] not set by perform_inline

### DISCUSSION THREAD

**Discussion by Yurokle** [sidekiq/sidekiq#6392] [General]
Hello. I've noticed that `job['enqueued_at']` isn't being set by `#perform_inline`. Is there a particular reason for it?
I thought it's supposed to work as an integration test and mimic `perform_async` behavior (which does set `job['enqueued_at']` unlike perform_in/_at).

**Comment by mperham** [comment]
Why would it be? It's never enqueued, it's executed inline.

**Comment by nateberkopec** [comment]
If it's an integration test, you should probably actually enqueue/drain the queue to test. Performing inline, if you don't do it in production, doesnt make sense in that context.
